src/components/ResourceList/tests/ResourceList.test.tsx

```diff
@@ -21,8 +21,14 @@ import {BulkActions, CheckableButton} from '../components';
 const itemsNoID = [{url: 'item 1'}, {url: 'item 2'}];
 const singleItemNoID = [{url: 'item 1'}];
 const singleItemWithID = [{id: '1', url: 'item 1'}];
+interface Item {
+  id: string;
+  name: string;
+  url: string;
+  title: string;
+}
 
-const itemsWithID = [
+const itemsWithID: Item[] = [
```

Comment:
Just curious if we needed this? The type should have been inferred :)

Reply:
That is true! That works.